resources/aws_ec2.go

```diff
@@ -708,3 +750,50 @@ func (obj *AwsEc2Res) snsPostHandler(w http.ResponseWriter, req *http.Request) {
 		log.Printf("%s: Post: %s", obj, string(post))
 	}
 }
+
+// snsMakeTopic creates a topic on aws sns.
+func (obj *AwsEc2Res) snsMakeTopic(snsClient *sns.SNS) (*string, error) {
+	// make topic
+	topicInput := &sns.CreateTopicInput{
+		Name: aws.String(SnsTopicName),
+	}
+	topic, err := snsClient.CreateTopic(topicInput)
+	if err != nil {
+		return nil, err
+	}
+	log.Printf("%s: Created SNS Topic", obj)
+	return topic.TopicArn, nil
```

Comment:
You've now created a scenario where you can return nil, nil. Is this what you want? It's legal, and exists in some situations, but typically that has a special meaning. What is it in this case? Will this cause a panic anywhere?